gtsfm/loader/one_d_sfm_loader.py

```diff
@@ -41,6 +41,7 @@ def __init__(
         folder: str,
         image_extension: str = "jpg",
         max_resolution: int = 640,
+        enable_no_exif: bool = False,
```

Comment:
Instead of the `enable_no_exif` and also a globally defined default focal length, I think it would make sense to have the default focal length as an attribute, and if it's None, then that should be interpreted as requiring EXIF.

Reply:
Done.
I kept it as float type and only non-positive value requires exif.  In my opinion it makes no sense to pass None to `get_intrinsics` in order to process images with no exif. Please help review when you have time. Thanks.

Reply:
I would prefer None over a negative number. It's more explicit, since focal lengths can actually be negative.

Reply:
I actually preferred `enable_no_exif` flag. A user is not going to know what the preferred value for the focal length factor is. It makes sense to have a reasonable default for it. And another flag to decide whether or not to use it.

Reply:
without `enable_no_exif`, please add some documentation on what values are recommended.

Reply:
I'm fine either way

Reply:
Thanks for the comments. Yeah, using a negative value to do the check is definitely a bad idea. I'll fix it.